Makefile

```diff
@@ -2,24 +2,33 @@ export GO15VENDOREXPERIMENT:=1
 export CGO_ENABLED:=0
 export GOARCH:=amd64
 
+LOCAL_OS:=$(shell uname | tr A-Z a-z)
```

Comment:
Could probably use native makefile globbing/functions instead. But mostly just easier to use well-understood utilities (find / grep / etc).